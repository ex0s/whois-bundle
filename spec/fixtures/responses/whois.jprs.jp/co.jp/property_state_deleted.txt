[ JPRS database provides information on network administration. Its use is    ]
[ restricted to network administration purposes. For further information,     ]
[ use 'whois -h whois.jprs.jp help'. To suppress Japanese output, add'/e'     ]
[ at the end of command, e.g. 'whois -h whois.jprs.jp xxx/e'.                 ]

Domain Information:
a. [Domain Name]                GAYLIFE.CO.JP
g. [Organization]               Suspended Domain Name
l. [Organization Type]          Suspended Domain Name
m. [Administrative Contact]      
n. [Technical Contact]          
p. [Name Server]                
s. [Signing Key]                
[State]                         Deleted (2013/01/31)
[Registered Date]                
[Connected Date]                 
[Last Update]                   2012/08/08 12:00:43 (JST)

