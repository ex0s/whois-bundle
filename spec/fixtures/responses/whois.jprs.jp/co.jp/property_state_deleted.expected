#status
  %s == :suspended

#available?
  %s == false

#registered?
  %s == true
